modules/dateDisplay.js

```diff
@@ -0,0 +1,16 @@
+// import { DateTime } from './luxon.js';
+
+const dateContainer = document.querySelector('.date');
+
+// const dateDisplay = () => {
+//   const date = DateTime.now().toString();
+//   dateContainer.innerHTML += date;
+// };
+
+// Using in-built function Date()
+const dateDisplay = () => {
+  const date = new Date();
+  dateContainer.innerHTML += date;
+};
```

Comment:
- [ ] As already pointed out by the previous reviewer, please kindly use the `luxon` date library instead of the native Date object as this is a core requirement.

Reply:
Kindly, read my comment on the previous PR, before you leave a review.